lib/tapioca/compilers/symbol_table_compiler.rb

```diff
@@ -140,6 +140,7 @@ def dispatch(event)
       sig { params(event: Gem::SymbolFound).void }
       def on_symbol(event)
         symbol = event.symbol
+        symbol = T.must(symbol[2..-1]) if symbol.start_with?("::")
```

Comment:
The `SymbolTableCompiler` was refactored recently but I think this is still roughly the equivalent change to: https://github.com/Shopify/tapioca/commit/fb329842c6a3ede2dd6447ad96cf4eb440b87ab1.